src/components/templates/WidgetsTemplate.tsx

```diff
@@ -57,7 +58,11 @@ const getWidgetByType = (widget: any) => {
     case 'accident_count_by_day_night': {
       widgetComponent = <CountAccidentsByDayNightPieWidget data={data} />;
       break;
-    }
+	}
+	case 'vision_zero_view': {
+		widgetComponent = <VisionZeroImageViewWidget data={data} />;
```

Comment:
Do not make it for a specific image (vision zero) - it should be for all static images